### DISCUSSION TITLE
ESlint

### DISCUSSION THREAD

**Discussion by r8bywork** [zed-industries/zed#9384] [Feature Requests]
Hi everyone!
How to customize so that eslint settings are taken from the root of the project? 
For example, the settings file - eslintrc.js - how to use settings from there? 
Because now, when I save the file, it is formatted in a chaotic way. 
But eslint project gives errors and says what to fix. 

```
"language_overrides": {
    "TSX": {
      "format_on_save": {
        "external": {
          // "command": "eslint",
          "command": "npx",
          "arguments": ["--stdin-filepath", "{buffer_path}", "--fix"]
        }
      }
    }
  }
 ```
 This is not working for tsx

**Comment by mrnugget** [comment]
Hey! Did you read the docs here? https://zed.dev/docs/javascript#eslint
